### ISSUE TITLE
Asset Tag ID doesn't autoincrement even though AutoIncrement is turned on

### ISSUE THREAD

**Issue by silicon11** [snipe/snipe-it#3498]
#### Expected Behavior (or desired behavior if a feature request)

Asset Tag ID Auto-Incrementing

-----

#### Actual Behavior

Asset Tag ID doesn't autoincrement even though AutoIncrement is turned on

-----

#### Please confirm you have done the following before posting your bug report:

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

-----
#### Please provide answers to these questions before posting your bug report:

- Version of Snipe-IT you're running: v3.6.4 build 12 (gf7e4fca) 
- What OS and web server you're running Snipe-IT on Ubuntu 16.04.2 LTS / Apache2
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc) install.sh
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error NO ERROR
- What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error
- If a stacktrace is provided in the error, include that too.
- Any errors that appear in your browser's error console. 
- Confirm whether the error is [reproduceable on the demo](https://snipeitapp.com/demo).
- Include any additional information you can find in `app/storage/logs` and your webserver's logs.
- Include what you've done so far in the installation, and if you got any error messages along the way.
- Indicate whether or not you've manually edited any data directly in the database

**Comment by snipe** [comment]
> What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error

More information would be helpful here. Are you creating a new asset, cloning an asset, etc?

**Comment by silicon11** [comment]
I have a prefix set it is "#-" without the quotes which i replace the # with a Letter before creating the asset. The "increment number" stays the same whether I am cloning or create a new asset, always number 95. Technically I should be on number 175 because that would be the next ID number for the assets table in the database.

**Comment by dmccabe-kinex** [comment]
I have the same issue. I have tried it with prefixes, without prefixes, and tried creating one manually, and then turning the Asset ID autoincrement back on. Still the same results. 

A little more detail for my situation:

I started with a clean install of 3.6.4. I imported a database from 3.6.2. Everything imported correctly, and the only issue that I have experienced was this one. I have it set for 5 digits, and it always tries to create asset 00691. With auto-incrementing, I was up to 01504.
